plugins/org.yakindu.base.xtext.utils.jface/src/org/yakindu/base/utils/jface/help/HelpHoverProvider.java

```diff
@@ -145,15 +144,18 @@ public IInformationControlCreator getInformationPresenterControlCreator() {
 	public class CustomPresenterControlCreator extends PresenterControlCreator {
 
 		protected void configureControl(IXtextBrowserInformationControl control, ToolBarManager tbm, String font) {
-			OpenInHelpAction openHelpAction = new OpenInHelpAction();
-			openHelpAction.setEnabled(true);
-			tbm.add(openHelpAction);
+			openExternalDocumentationAction.setEnabled(true);
+			tbm.add(openExternalDocumentationAction);
 			tbm.update(true);
 		};
 
 	}
 
-	protected static final class OpenInHelpAction extends Action {
+	public static final class OpenInHelpAction extends Action {
+
+		@Inject(optional = true)
+		@Named("openhelp.contextid")
+		protected String helpKeyword;
```

Comment:
* let's rename helpKeyword->contextId
* let's put "openhelp.contextid" into a constant than we can access it in the modules. That is better than copying strings.